docs/products/clickhouse/howto/check-data-tiered-storage.rst

```diff
@@ -0,0 +1,81 @@
+
+Prerequisites
+-------------
+
+* Aiven organization
+* Access to `Aiven Console <https://console.aiven.io/>`_
+* Tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on an Aiven for ClickHouse service level and on a table level
+* Command line tool (:doc:`ClickHouse client </docs/products/clickhouse/howto/connect-with-clickhouse-cli>`)
+
+Check data distribution in Aiven Console
+----------------------------------------
+
+You can use `Aiven Console <https://console.aiven.io/>`_ to check if the tiered storage is enabled on your service and, if so, how much storage has been used for each type of the disks (the local SSD and the remote object storage) by particular tables.
+
+To check the tiered storage's status information, go to `Aiven Console <https://console.aiven.io/>`_ > your Aiven for ClickHouse service > the **Overview** page > the **Tiered storage** section.
+
+Run a data distribution check with the ClickHouse client (CLI)
+--------------------------------------------------------------
+
+1. :doc:`Connect to your Aiven for ClickHouse service </docs/products/clickhouse/howto/list-connect-to-service>` using, for example, the ClickHouse client (CLI).
+2. Run the following query:
+
+   .. code-block:: bash
+
+    SELECT
+        database,
+        table,
+        disk_name,
+        formatReadableSize(sum(data_compressed_bytes)) AS total_size,
+        count(*) AS parts_count,
+        formatReadableSize(min(data_compressed_bytes)) AS min_part_size,
+        formatReadableSize(median(data_compressed_bytes)) AS median_part_size,
+        formatReadableSize(max(data_compressed_bytes)) AS max_part_size
+    FROM system.parts
+    GROUP BY
+        database,
+        table,
+        disk_name
+    ORDER BY
+        database ASC,
+        table ASC,
+        disk_name ASC
+
+   You can expect to receive the following output:
+
+   .. code-block:: bash
+
+    ┌─database─┬─table─────┬─disk_name─┬─total_size─┬─parts_count─┬─min_part_size─┬─median_part_size─┬─max_part_size─┐
+    │ datasets │ hits_v1   │ default   │ 1.20 GiB   │           6 │ 33.65 MiB     │ 238.69 MiB       │ 253.18 MiB    │
+    │ datasets │ visits_v1 │ S3        │ 536.69 MiB │           5 │ 44.61 MiB     │ 57.90 MiB        │ 317.19 MiB    │
+    │ system   │ query_log │ default   │ 75.85 MiB  │         102 │ 7.51 KiB      │ 12.36 KiB        │ 1.55 MiB      │
+    └──────────┴───────────┴───────────┴────────────┴─────────────┴───────────────┴──────────────────┴───────────────┘
+
+.. topic:: Result
+   
+   The query returns a table with data distribution details for all databases and tables that belong to your service: storage device they use, their total sizes as well as parts' counts and sizing.
```

Comment:
... *the* storage device they use ....  ?
I'd also drop the possessive apostrophe on "parts"